### ISSUE TITLE
Do full hyperparameter search

### ISSUE THREAD

**Issue by ikmckenz** [ikmckenz/target-pred-py#28]
After issue #27 is resolved, make a new jupyter notebook to conduct a full hyperparameter search to find the best defaults. Should also test different loss functions, network activations, etc.
